common/src/main/java/com/wynntils/handlers/item/ItemHandler.java

```diff
@@ -149,15 +147,22 @@ private String getBaseName(String name) {
         return bracketIndex == -1 ? name : name.substring(0, bracketIndex);
     }
 
-    private boolean similarStack(ItemStack firstItem, ItemStack secondItem) {
-        if (!firstItem.getItem().equals(secondItem.getItem())) return false;
+    private boolean sameGearWithSkin(ItemStack existingItem, ItemStack newItem) {
+        if (WynnItemMatchers.isGearSkin(existingItem) || !WynnItemMatchers.isGearSkin(newItem)) return false;
```

Comment:
Hm, I realize now that this line is strictly not necessary. What the code below checks is really "is the new lore identical to the old, except that it has an added line". That is a more general approach.

Sorry for all the confusing comments in random order here, but I think this approach is actually quite good, but you can drop the "isGearSkin" test and just dive directly into comparing the lore. You can also start by checking not only size inequality, but if new is exactly one line longer than existing, otherwise abort quickly.

And name the method not `sameGearWithSkin` but like `isLoreAlmostEquals` (and maybe move it to LoreUtils?)

Reply:
... and then you can actually replace the original call to `isLoreEquals` with `isLoreAlmostEquals`, you don't need to check this twice.

Reply:
This is a great point actually! I don't know off the top of my head if there are any other instances where lore would get "added to" like this, but it's probably safe to assume that if they exist, they can be handled in the same way. I will try to implement this later today

Reply:
Yes, if this kind of behavior would appear anywhere else, I think it will be reasonable to consider it the "same" item.